### DISCUSSION TITLE
Feature no longer present: Download/Delete Models

### DISCUSSION THREAD

**Discussion by Aleprex** [open-webui/open-webui#7391] [General]
# Bug Report

---

## Installation Method

Docker 

## Environment

- **Open WebUI Version:** [0.4.5]
- **Ollama :** [0.4.4]

- **Operating System:** [Kubuntu 22.04]
- **Browser  :** [Brave [Versione 1.73.91 Chromium: 131.0.6778.85]]

**Confirmation:**

- [ X] I have read and followed all the instructions provided in the README.md.
- [ X] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [ X] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Description
In the latest versions there is no longer the possibility to download new models or import GGUF files, and there is no longer even the possibility to delete individual models, but only "Delete All Models".

**Screenshots/Screen Recordings (if applicable):**
![image](https://github.com/user-attachments/assets/2264d902-0d87-4921-8d35-b852d8eccdac)

## Old features
 📥🗑️ Download/Delete Models: Models can be downloaded or deleted directly from Open WebUI with ease
![image](https://github.com/user-attachments/assets/1e8ac1d8-f799-43a4-9ca5-83681e78e94d)

**Comment by sebaxakerhtc** [comment]
@Aleprex This feature was moved.
![изображение](https://github.com/user-attachments/assets/daa5fac3-b5e5-4f82-b780-a097f7bd0a9a)

**Reply by Aleprex** [reply]
Thank you very much!

**Reply by MattariOnline** [reply]
Thank you VERY much!

I feel they should add a footnote in the models tab telling users how it is done now; it is _incredibly_ confusing for someone who has used the previous system which seems far more straight forward.

I get why the change was made, and it makes sense, but I feel it'd have been better to keep the old pull/delete model section and just add an option to choose which connection it affects.
